assets/training/finetune_acft_common/components/validation/spec.yaml

```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 type: command
 
-version: 0.0.7
+version: 0.0.8
 name: finetune_common_validation
 display_name: Common Validation Component
 description: Component to validate the finetune job against Validation Service
 
 is_deterministic: True
 
-environment: azureml://registries/azureml/environments/acpt-pytorch-2.2-cuda12.1/versions/23
+environment: azureml://registries/azureml/environments/acpt-pytorch-2.2-cuda12.1/labels/latest
```

Comment:
Setting to latest is good because it means we can just release the component without code changes, but bad because it makes it slightly harder to figure out which component version was last released/will be released next. I prefer the current format (explicit version); want to discuss pros and cons?

Reply:
Discussed outside of PR. It's manageable for the vision team to update versions but tedious and error prone for a single person (eg the DRI) to update versions for all the components (vision, nlp, forecasting). Agreeing with the labels/latest approach.